dev/conductor/ui/lib/main.dart

```diff
@@ -2,24 +2,35 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io' as io;
 import 'package:conductor_core/conductor_core.dart';
 import 'package:conductor_core/proto.dart' as pb;
-
+import 'package:desktop_window/desktop_window.dart';
 import 'package:file/file.dart';
 import 'package:file/local.dart';
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:platform/platform.dart';
 
-const String _title = 'Flutter Conductor';
+import 'widgets/progression.dart';
+
+const String _title = 'Flutter Desktop Conductor (Not ready, do not use)';
 
 const LocalFileSystem _fs = LocalFileSystem();
 const LocalPlatform _platform = LocalPlatform();
 final String _stateFilePath = defaultStateFilePath(_platform);
 
-void main() {
+Future<void> main() async {
   final File _stateFile = _fs.file(_stateFilePath);
-  final pb.ConductorState? state = _stateFile.existsSync() ? readStateFromFile(_stateFile) : null;
-  runApp(MyApp(state));
+  final pb.ConductorState? state =
+      _stateFile.existsSync() ? readStateFromFile(_stateFile) : null;
+
+  WidgetsFlutterBinding.ensureInitialized();
+  // app currently only supports macOS and Linux, and a minimum app size is added to prevent overflow
+  if (!kIsWeb && (io.Platform.isMacOS || io.Platform.isLinux)) {
+    await DesktopWindow.setMinWindowSize(const Size(600, 600));
```

Comment:
If we're failing early, this shouldn't be necessary (the default size should be ok)